cl/assets/react/disclosure-page.css

```diff
@@ -0,0 +1,33 @@
+/*Disclosure Table Viewer CSS*/
+
+.fa-exclamation-triangle {
+  font-size: 100px;
+}
+
+.search-panel-td {
+  border-bottom: 0.5px #cecece solid;
+}
+
+.search-panel-table {
+  width: 100%
+}
+
+
+/*Disclosure List Homepage CSS*/
+
+.tr-results {
+  border-bottom: 1px solid black;
+  padding: 10px;
+  /*background: rgba(255, 255, 255, 0.98);*/
+  width: 100%;
+
+}
+
+.table-instant-results {
+  width: 95%;
+  border-left: 1px solid;
+  border-right: 1px solid;
+  position: absolute;
+  background: rgba(255, 255, 255, 0.95);
```

Comment:
Transparency? Let's use solid white.

Reply:
I see, my dreams of frosted glass are for not.  Sounds good.